tools/topology/topology2/cavs-sdw.conf

```diff
@@ -69,6 +69,11 @@ Define {
 	DEEP_BUFFER_PIPELINE_SRC        'mixin.15.1'
 	DEEP_BUFFER_PIPELINE_SINK       'mixout.1.1'
 	DEEP_BUFFER_PCM_NAME		'Deepbuffer Jack Out'
+	DEEP_BUFFER_PIPELINE_ID_2	16
+	DEEP_BUFFER_PCM_ID_2		35
+	DEEP_BUFFER_PIPELINE_SRC_2	'mixin.16.1'
+	DEEP_BUFFER_PIPELINE_SINK_2	'mixout.21.1'
+	DEEP_BUFFER_PCM_NAME_2		'Deepbuffer Amps'
```

Comment:
@ranj063 @naveen-manohar should this be "Speaker" instead of "Amps" ?

Reply:
@lgirdwood @ranj063: I noticed a uniformity throughout for 'Deepbuffer Amps'

> development/mtl-sdw-hdmi.conf:83: DEEP_BUFFER_PCM_NAME_2 'Deepbuffer Amps'
> development/cavs-sdw-hdmi.conf:83: DEEP_BUFFER_PCM_NAME_2 'Deepbuffer Amps'
> development/cavs-sdw.conf:83: DEEP_BUFFER_PCM_NAME_2 'Deepbuffer Amps'

Please confirm if I need to modify to 'Deepbuffer Speaker' ?

Reply:
Good point - this may change UCM too, but the users today are all in "development" folder so are not used in production. @naveen-manohar I think we should change all to "Speaker" for clarity. @plbossart @ranj063 @singalsu anyone disagree ?